src/main/java/seedu/trackbeau/logic/parser/FindCommandParser.java

```diff
@@ -1,85 +1,96 @@
 package seedu.trackbeau.logic.parser;
 
 import static seedu.trackbeau.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static seedu.trackbeau.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
+import static seedu.trackbeau.logic.parser.CliSyntax.*;
```

Comment:
I think can change it to explicitly state all the classes you are importing to adhere to the coding guidelines :)

Reply:
Fix in previous commit